docs/mkDocs/docs/options.md

```diff
@@ -733,3 +733,52 @@ an import of a module coming from package `PACKAGE`. e.g.
 `--exclude-automatic-assumptions-for=vector` would stop loading
 `_LHAssumptions` modules for any imports coming from package
 `vector`.
+
+## Warn on Term Holes
+
+**Options:** `warn-on-term-holes`
+
+LiquidHaskell can be configured to emit warnings whenever it encounters incomplete terms, known as *holes*.
```

Comment:
This is probably a good place to explain what a hole or an incomplete term is. Otherwise, I don't think the user will know that she needs to define variables named "hole".